pkg/wwan/Dockerfile

```diff
@@ -48,6 +49,14 @@ WORKDIR /picocom
 # Need this patch to build with musl: https://github.com/npat-efault/picocom/commit/1acf1ddabaf3576b4023c4f6f09c5a3e4b086fb8
 RUN make -j "$(getconf _NPROCESSORS_ONLN)" && strip picocom && cp picocom /usr/bin/
 
+# Install FCC-unlock scripts/tools provided by device vendors
+ADD --keep-git-dir=true https://github.com/lenovo/lenovo-wwan-unlock.git#${LENOVO_WWAN_UNLOCK_COMMIT} /lenovo-wwan-unlock
+WORKDIR /lenovo-wwan-unlock
+RUN cp libmbimtools.so /out/usr/lib/ && chmod 444 /out/usr/lib/libmbimtools.so
```

Comment:
I think you need to pack both RUNs into one to make yetus happy...

Reply:
Where do you see this yetus error? `results-full.txt` in `yetus-scan.zip` is empty even though the yetus check failed.

Reply:
I re-run the action, actually errors were in shellcheck:

```
pkg/wwan/fcc-unlock/1eac:1002:14:89: note: Note that A && B || C is not if-then-else. C may run when A is true. [SC2015]
pkg/wwan/fcc-unlock/2c7c:030a:14:89: note: Note that A && B || C is not if-then-else. C may run when A is true. [SC2015]
pkg/wwan/fcc-unlock/2c7c:0311:14:89: note: Note that A && B || C is not if-then-else. C may run when A is true. [SC2015]
```

https://github.com/lf-edge/eve/actions/runs/9082764731/artifacts/1505442469

Reply:
> Note that A && B || C is not if-then-else. C may run when A is true

But that is exactly how I want it to behave :)
If the Lenovo tool fails, try the open-source solution as a fallback.

Reply:
Then put in the comment(s) to ignore SC2015

Reply:
Done